src/Adapter/Order/CommandHandler/UpdateProductInOrderHandler.php

```diff
@@ -186,4 +217,164 @@ private function assertProductCanBeUpdated(
             }
         }
     }
+
+    /**
+     * @param Number $priceTaxIncluded
+     * @param Number $priceTaxExcluded
+     * @param Order $order
+     * @param Cart $cart
+     * @param Product $product
+     * @param Combination|null $combination
+     */
+    private function createSpecificPriceIfNeeded(
```

Comment:
This code is duplicated with AddProductToOrder maybe it should be in the abstract class or in a trait, or in a service This would avoid fixing bugs in 4 places each time